pylab/website/forms.py

```diff
@@ -54,16 +55,33 @@ def clean(self):
             self.check_existing_events(title, starts)
 
 
-class VotePointsForm(forms.ModelForm):
+class ProjectPointsForm(forms.ModelForm):
+    points = forms.IntegerField(
+        widget=forms.NumberInput(attrs={'max': 99, 'min': 0, 'class': 'vote-points-input'}),
+        validators=[MinValueValidator(0)],
```

Comment:
Instead of validator you can use `min_value` attribute: https://docs.djangoproject.com/en/1.8/ref/forms/fields/#django.forms.IntegerField.max_value